oceannavigator/frontend/src/components/OceanNavigator.jsx

```diff
@@ -134,8 +135,10 @@ class OceanNavigator extends React.Component {
   }
 
   // Turns off map drawing
-  removeMapInteraction(mode) {
-    this.mapComponent.removeMapInteractions(mode);
+  removeMapInteraction(mode="All") {
```

Comment:
```suggestion
  removeMapInteraction(mode = "All") {
```

Reply:
Can `mode` be renamed to be something more descriptive? I don't understand what it means.